AirCasting/SearchAndFollow/CompleteScreen/Views/CompleteScreenViewModel.swift

```diff
@@ -42,11 +43,11 @@ class CompleteScreenViewModel: ObservableObject {
     let sessionStartTime: Date
     let sessionEndTime: Date
     let sensorType: String
-    @Published var completeButtonEnabled: Bool = false
-    @Published var completeButtonText: String = Strings.CompleteSearchView.confirmationButtonTitle
+    @Published var followButtonEnabled: Bool = false
+    @Published var followButtonText: String = Strings.CompleteSearchView.followButtonTitle
```

Comment:
if you could group those `@Published` together it would be great, I know it something additional so if not please skip